server/modules/storage/digitalocean/definition.yml

```diff
@@ -16,7 +16,6 @@ props:
     hint: The DigitalOcean datacenter region where the Space will be created.
     default: nyc3
     enum:
-      - ams3
```

Comment:
Not sure if `ams3` should be removed until `ams3` is fully available again, currently you can't create new space in this region and existing space will be rate limited: https://www.digitalocean.com/docs/platform/release-notes/2019/spaces-ams3/

Reply:
I would leave it in, as this is a temporary issue and some users might already be using a space in this region.